recipe/build-dali-tf-plugin.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# *****************************************************************
+# (C) Copyright IBM Corp. 2019, 2021. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *****************************************************************
+
+# Determine Architecture
+
+ARCH="$(arch)"
+if [ ${ARCH} = "x86_64" ]; then
```

Comment:
I thought our x86 toolchain pipeline is cos7 and ppc is cos6 ?

Reply:
No, it's other way round. ppc is cos7 and x86 was cos6 which we updated in 1.4.0.